### PR TITLE
Makes redirect_uri and logout_redirect_uri configurable

### PR DESCRIPTION
**Pull request by psignoret**
Closes issue #57

### CODE REVIEW

File: SettingsPage.php [hunk 1]
```diff
@@ -339,7 +362,40 @@ public function client_secret_callback() {
...
+		);
+	}
+
+	/**
+	 * Renders the `redirect_uri` form control
+	 **/
+	public function redirect_uri_callback() {
+		$this->render_text_field( 'redirect_uri' );
+		printf(
+			' <a href="#" onclick="jQuery(\'#redirect_uri\').val(\'%s\'); return false;">%s</a>'
```
Comment (bradkovach) [inline SettingsPage.php]: should jQuery be added to an external, queued javascript file?

Comment (bradkovach) [inline SettingsPage.php]: maybe implement a pattern of `<a href="#" class="reset-trigger">` and then `<input class="resettable" data-default="%s">`

File: SettingsPage.php [hunk 2]
```diff
@@ -372,20 +428,20 @@ public function field_to_match_to_upn_callback() {
...
+
 		// Default configuration should be most-benign
 		if( ! isset( $this->settings['default_wp_role'] ) ) {
 			$this->settings['default_wp_role'] = '';
 		}
-		
+
 		echo '<select name="aadsso_settings[default_wp_role]" id="default_wp_role">';
-		printf( '<option value="%s">%s</option>', '', '(None, deny access)' ); 
+		printf( '<option value="%s">%s</option>', '', '(None, deny access)' );
```
Comment (bradkovach) [inline SettingsPage.php]: This option sprintf template is used frequently, should it be placed in a variable for reuse?

File: SettingsPage.php (hunk 1 above)
Comment (psignoret) [inline SettingsPage.php]: If there are more of these, sure. But the only "useful" default values are these two, at the moment.

File: SettingsPage.php (hunk 2 above)
Comment (psignoret) [inline SettingsPage.php]: This specific pattern is only used once, though a slightly more complex one could be used below this, and in `field_to_match_to_upn_callback`. Next time someone's poking around that they can certainly change it.

**Pull request merged**